lib/signs_ui/config/state.ex

```diff
@@ -71,16 +72,22 @@ defmodule SignsUi.Config.State do
 
   @spec init(any()) :: {:ok, t()} | {:stop, any()}
```

Comment:
Minor: we can remove `{:stop, any()}` from the typespec since we'll never return that anymore